source/StigData/Processed/FireFox-All-5.2.xml

```diff
@@ -153,163 +191,152 @@ Criteria: If the parameter is set incorrectly, then this is a finding.
 If the setting is not locked, this is a finding.</RawString>
       <Value>false</Value>
     </Rule>
-    <Rule id="V-19742" severity="medium" conversionstatus="pass" title="DTBF090-Firefox Preferences-Addons\ plugin updates" dscresource="ReplaceText">
-      <Description>&lt;VulnDiscussion&gt;Set this to false to disable checking for updated versions of the Extensions/Themes.  Automatic updates from untrusted sites puts the enclave at risk of attack and may override security settings.&lt;/VulnDiscussion&gt;&lt;FalsePositives&gt;&lt;/FalsePositives&gt;&lt;FalseNegatives&gt;&lt;/FalseNegatives&gt;&lt;Documentable&gt;false&lt;/Documentable&gt;&lt;Mitigations&gt;&lt;/Mitigations&gt;&lt;SeverityOverrideGuidance&gt;&lt;/SeverityOverrideGuidance&gt;&lt;PotentialImpacts&gt;&lt;/PotentialImpacts&gt;&lt;ThirdPartyTools&gt;&lt;/ThirdPartyTools&gt;&lt;MitigationControl&gt;&lt;/MitigationControl&gt;&lt;Responsibility&gt;System Administrator&lt;/Responsibility&gt;&lt;IAControls&gt;&lt;/IAControls&gt;</Description>
+    <Rule id="V-223167.a" severity="medium" conversionstatus="pass" title="SRG-APP-000141" dscresource="ReplaceText">
```

Comment:
This Needs to be reevaluated.. The STIG says xpinstalled = "false" or "locked" or DisableSystemAddonUpdate = 1.. we need to test manually setting these values then confirm we are setting them correctly

Reply:
Also, the ".a" and ".b" rule parsed the same.

Reply:
This rule can be set two different ways, 
1. a configuration file
2. Group Policy

If you use a configuration file, you can validate the setting in the browser (about:config), otherwise if a GPO was used then you must check the registry: 

V-223167.c::*::HardCodedRule(RegistryRule)@{DscResource = 'RegistryPolicyFile'; Key = '\HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Mozilla\Firefox'; ValueData = '1'; ValueName = 'DisableSystemAddonUpdate'; ValueType = 'Dword'}

Reply:
The .a and .b rule are still the same.

Reply:
I need help with correcting this one.